### PR TITLE
AZ translation for 4.2

### PR DESCRIPTION
**Pull request by kerimovscreations**
#15

### CODE REVIEW

File: 4-Classification/2-Classifiers-1/translations/README.az.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+    ```python
+    from sklearn.linear_model import LogisticRegression
+    from sklearn.model_selection import train_test_split, cross_val_score
+    from sklearn.metrics import accuracy_score,precision_score,confusion_matrix,classification_report, precision_recall_curve
+    from sklearn.svm import SVC
+    import numpy as np
+    ```
+
+1. X və y kordinatlarını öyrənmə üçün iki data qrupuna ayırın. `cuisine` data qrupunun adı ola bilər:
```
Comment (aliyevakhalida) [inline 4-Classification/2-Classifiers-1/translations/README.az.md:43]: typo:
```suggestion
1. X və y koordinatlarını öyrənmə üçün iki data qrupuna ayırın. `cuisine` data qrupunun adı ola bilər:
```

File: 4-Classification/2-Classifiers-1/translations/README.az.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
+# Mətbəx təsnifatlandıcıları - hissə 1
+
+Bu dərsdə əvvəlki dərsdən əldə etdiyimiz balanslı və təmizlənmiş mətbəx data massivindən istifadə edəcəksiniz.
+
+Bu data massivini müxtəlix təsnifatlandıcılarda istifadə edərək _inqredientlər əsasında milli mətbəxləri təxmin edəcəksiniz_. Bunu edərkən alqoritmi başqa hansı təsnifatlandırma tapşırıqlarında istifadə edə biləcəyinizi də öyrənəcəksiniz.
+
+## [Mühazirədən əvvəl test](https://gray-sand-07a10f403.1.azurestaticapps.net/quiz/21/)
+# Hazırlıq
+
+Sizin [1-ci dərsi](../../1-Introduction/translations/assignment.az.md) bitirdiyinizi təxmin edirik. Əlavə olaraq bu 4 dərsdə istifadə edəcəyimiz _cleaned_cuisines.csv_ sənədinin `/data` qovluğunda mövcud olduğuna əmin olun.
```
Comment (muradtries) [inline 4-Classification/2-Classifiers-1/translations/README.az.md:10]:
```suggestion
Sizin [1-ci dərsi](../../1-Introduction/translations/assignment.az.md) bitirdiyinizi güman edirik. Əlavə olaraq bu 4 dərsdə istifadə edəcəyimiz _cleaned_cuisines.csv_ faylının `/data` qovluğunda mövcud olduğuna əmin olun.
```

File: 4-Classification/2-Classifiers-1/translations/README.az.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+- **Gaussian Processes** (Qauss emalları)
+- **Decision Trees** (Qərar sxemləri)
+- **Ensemble methods (voting Classifier)** Ansambl üsullar (səsvermə əsaslı təsnifatlandırma)
+- Çoxsaylı etiket və çoxsaylı çıxış alqoritmləri (multiclass and multilabel classification, multiclass-multioutput classification)
+
+> Siz həmçinin [təsnifatlandırma üçün neyron şəbəkələr](https://scikit-learn.org/stable/modules/neural_networks_supervised.html#classification) də istifadə edə bilərsiniz, lakin bu mövzu bizim dərsimizdən kənardır.
+
+### Hansı təsnifatlandırıcı seçilməlidir?
+
+İndi siz hansı təsnifatlandırıcı seçməlisiniz? Adətən bunun cavabını bir çox üsulu yoxlamaq və daha yaxşı nəticə göstərəni seçməklə tapmaq olur. Yaratdığınız data qruplarını **Scikit-learn** kitabxanasıda [yanbayan müqayisə](https://scikit-learn.org/stable/auto_examples/classification/plot_classifier_comparison.html) edə bilərsiniz. Aşağıda nümunə kimi KNeighbors, SVC two ways, GaussianProcessClassifier, DecisionTreeClassifier, RandomForestClassifier, MLPClassifier, AdaBoostClassifier, GaussianNB and QuadraticDiscrinationAnalysis alqoritmləri müqayisəsi göstərilib:
```
Comment (muradtries) [inline 4-Classification/2-Classifiers-1/translations/README.az.md:99]:
```suggestion
İndi siz hansı təsnifatlandırıcı seçməlisiniz? Adətən bunun cavabını bir çox üsulu yoxlamaq və daha yaxşı nəticə göstərəni seçməklə tapmaq olur. Yaratdığınız data qruplarını **Scikit-learn** kitabxanasında [yanbayan müqayisə](https://scikit-learn.org/stable/auto_examples/classification/plot_classifier_comparison.html) edə bilərsiniz. Aşağıda nümunə kimi KNeighbors, SVC two ways, GaussianProcessClassifier, DecisionTreeClassifier, RandomForestClassifier, MLPClassifier, AdaBoostClassifier, GaussianNB and QuadraticDiscrinationAnalysis alqoritmləri müqayisəsi göstərilib:
```

File: 4-Classification/2-Classifiers-1/translations/README.az.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+Sənədlərə əsasən `multi_class` seçimində aşağıdakı öyrənmə alqoritmləri seçilə bilər:
+
+- **one-vs-rest (OvR) sxemi istifadə edənlər**, əgər `multi_class` parametri `ovr` seçilmişdirsə.
+- **cross-entropy loss (çarpaz-entropiya itkisi) istifadə edənlər**, əgər `multi_class` parametri `multinomial` seçilmişdirsə. (Hazırda `multinomial` seçimi yalnız ‘lbfgs’, ‘sag’, ‘saga’ və ‘newton-cg’ alqoritmləri tərəfindən dəstəklənir)"
+
+> 🎓 Burada 'scheme' parametri 'ovr' (one-vs-rest) və ya 'multinomial' ola bilər. Logistik reqressiya əslində binary (ikili) təsnifatlandırma üçün tərtib olunduğu üçün bu sxemlər çoxsaylı etiket təsnifatlandırma tapşırıqları üçün daha uyğundur. [mənbə](https://machinelearningmastery.com/one-vs-rest-and-one-vs-one-for-multi-class-classification/)
+
+> 🎓 'solver' parametri "optimizasiya üçün istifadə olunacaq alqoritmi" nəzərdə tutur. [mənbə](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html?highlight=logistic%20regressio#sklearn.linear_model.LogisticRegression).
+
+Aşağıdakı cədvəldə Scikit-learn kitabxanasındakı alqoritmlərlə müxtəlif növ tapşırıqları və data strukturlarını necə idarə edə biləcəyininiz göstərilib:
```
Comment (muradtries) [inline 4-Classification/2-Classifiers-1/translations/README.az.md:139]:
```suggestion
Aşağıdakı cədvəldə Scikit-learn kitabxanasındakı alqoritmlərlə müxtəlif növ tapşırıqları və data strukturlarını necə idarə edə biləcəyiniz göstərilib:
```

File: 4-Classification/2-Classifiers-1/translations/README.az.md [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+
+    accuracy = model.score(X_test, y_test)
+    print ("Accuracy is {}".format(accuracy))
+    ```
+
+    ✅ Başqa bir alqoritmi seçib yoxlayın, misal üçün `lbfgs`.
+
+    > Qeyd. Pandas kitabxanasında [`ravel`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.ravel.html) funksiyasını istifadə edərək data modelinizi daha səthi (az-ölçülü) formaya keçirə bilərsiniz.
+
+    Dəqiqlik **80%**-dən yuxarı olması yaxşıdır!
```
Comment (muradtries) [inline 4-Classification/2-Classifiers-1/translations/README.az.md:169]:
```suggestion
    Dəqiqliyin **80%**-dən yuxarı olması yaxşıdır!
```

File: 4-Classification/2-Classifiers-1/translations/README.az.md [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+    Nəticə belə çap olunacaq:
+
+   ```output
+   ingredients: Index(['cilantro', 'onion', 'pea', 'potato', 'tomato', 'vegetable_oil'], dtype='object')
+   cuisine: indian
+   ```
+
+   ✅ Fərqli sıra nömrələrini yoxla və nəticəyə bax.
+
+1. Daha dərinə getsək təsnifat modelinin dəqiqliyini belə yoxlaya bilərsiniz:
```
Comment (muradtries) [inline 4-Classification/2-Classifiers-1/translations/README.az.md:187]:
```suggestion
1. Daha dərinə getsək, təsnifat modelinin dəqiqliyini belə yoxlaya bilərsiniz:
```

File: 4-Classification/2-Classifiers-1/translations/README.az.md [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+    | japanese     | 0.70      | 0.75   | 0.72     | 220     |
+    | korean       | 0.86      | 0.76   | 0.81     | 242     |
+    | thai         | 0.79      | 0.85   | 0.82     | 254     |
+    | accuracy     | 0.80      | 1199   |          |         |
+    | macro avg    | 0.80      | 0.80   | 0.80     | 1199    |
+    | weighted avg | 0.80      | 0.80   | 0.80     | 1199    |
+
+## 🚀Məşğələ
+
+Bu dərsdə siz təmizlənmiş data istifadə edərək inqredientlər əsasında milli mətbəxi təxmin edə biləcək maşın öyrənmə modelini qurdunuz. Scikit-learn kitabxanası istifadə etməklə daha hansı üsullarla təsnifatlandırma etmək mümkün olduğunu oxuyun. "Solver (alqoritm)" anlayışı üzərində daha dərinə gedərək arxa planda necə işlədiyini öyrənin.
```
Comment (muradtries) [inline 4-Classification/2-Classifiers-1/translations/README.az.md:231]:
```suggestion
Bu dərsdə siz təmizlənmiş data istifadə edərək inqrediyentlər əsasında milli mətbəxi təxmin edə biləcək maşın öyrənməsi modelini qurdunuz. Scikit-learn kitabxanası istifadə etməklə daha hansı üsullarla təsnifatlandırma etmək mümkün olduğunu oxuyun. "Solver (alqoritm)" anlayışı üzərində daha dərinə gedərək arxa planda necə işlədiyini öyrənin.
```

File: 4-Classification/2-Classifiers-1/translations/assignment.az.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
+# Təsnifatlandırıcı alqoritmlərini araşdır
+
+## Təlimatlar
+
+Bu dərsdə siz təsnifatlandırıcı alqoritmlərinin maşın öyrənmə alqoritmləri ilə necə birlikdə dəqiq modellər qurulmasını öyrəndiniz. Təsnifatlandırma alqoritmləri üzərindən keçin və ikisini seçin. Öz sözlərinizlə bu alqoritmləri bir-biri ilə müqayisə edin. Hansı növ problemləri həll etməyə nəzərdə tutulublar? Müxtəlif data strukturları ilə necə işləyirlər? Birini digəri əvəzinə nə üçün seçərsən?
```
Comment (muradtries) [inline 4-Classification/2-Classifiers-1/translations/assignment.az.md:5]:
```suggestion
Bu dərsdə siz təsnifatlandırıcı alqoritmlərinin maşın öyrənməsi alqoritmləri ilə necə birlikdə dəqiq modellər qurulmasını öyrəndiniz. Təsnifatlandırma alqoritmləri üzərindən keçin və ikisini seçin. Öz sözlərinizlə bu alqoritmləri bir-biri ilə müqayisə edin. Hansı növ problemləri həll etməyə nəzərdə tutulublar? Müxtəlif data strukturları ilə necə işləyirlər? Birini digəri əvəzinə nə üçün seçərsən?
```

**Review by muradtries — APPROVED** [review]

**Comment by kerimovscreations** [comment]
@muradtries pls commit your proposed changes into this branch, later let's merge this

**Comment by muradtries** [comment]
> @muradtries pls commit your proposed changes into this branch, then let's merge this

✅
